### ISSUE TITLE
No app after respring

### ISSUE THREAD

**Issue by ghost** [netsirkl64/palera1n-High-Sierra#10]
iPhone8+ 16.2 macOS 10.13.6

**Comment by ghost** [comment]
as of
5:41 PM CET (UTC+1)
Monday, December 26, 2022

https://github.com/netsirkl64/palera1n-High-Sierra/releases/

there has been a new release made to fix this exact issue
it was a known issue where `uicache` bin was getting exit code 1 and failing to trigger `uikittools`
`uicache` is required for palera1n to appear on home screen on ios 16

there was also another issue where the line endings in post.sh were CRLF instead of LF
the line endings have to be LF for the script to run properly on ios
this was causing the entire post.sh script to fail to run which also meant palera1n would not show on ios 16

both of these issues have been fixed

you will need to remove palera1n.app off of your mac os computer
and reboot your computer, and download the updated version
and restorerootfs under options in the app and then re jailbreak

then it should work

please reply once you have done this and it worked
then i will close as solved

**Comment by ghost** [comment]
Still no app after deleting rebooting and redownloading Mac app, then rejailbreak and respring iPhone, nothing in settings

**Comment by ghost** [comment]
after phase 1 is done it will need you to hit start again a second time in the gui, did you do that?

**Comment by ghost** [comment]
The app appeared just before respring, so I ran jb again, opened palera1n app hit install, but then respring froze for 10 mins, I forced reboot, have Cydia now but it won't open after rejailbreaking, you can close issue I'll try again after a restore

**Comment by ghost** [comment]
i will keep issue open as i have found another issue related to this. mobile phone app and mobile sms app are having issues after this update. i will investigate further and keep this thread posted.

**Comment by ghost** [comment]
Just tried your Mac app, installed palera1n and it was still there after the respiring, opened app hit install, doesn't install anything then resprings ![image](https://user-images.githubusercontent.com/1918482/209653481-a0e1ced8-eae5-47f9-a423-765a41592ad6.png)

**Comment by KirkLandTrip** [comment]
> Just tried your Mac app, installed palera1n and it was still there after the respiring, opened app hit install, doesn't install anything then resprings 

I have this problem as well.  
I *just* downlaoded the code from this repo. Initially I tried by running:  
`.\palera1n.sh --tweaks 16.0.3 --semi-tethered`  
But I also tried the GUI version, from the Releases section.

I have an iPhone X, with iOS 16.0.3

In both cases the device booted after the exploit, 30 seconds later a respring occurred & the palera1n app appeared.  
I tried clicking "Install" in the app and it wrote both of those lines:
```
[*]  Starting bootstrap process
[*]  Preparing bootstrap
```
Then it got stuck for a few seconds and the app "crashed" (?) -- by that I mean the GUI disappeared and instead I saw my iPhone's background with a darkish-transparent overlay above it.
If I try to go to the home screen I see that the  palera1n app is gone (both from home screen & then apps search)

**Comment by ktulufhtagn** [comment]
I still have this problem, after jb(tethered), i don't see the app(even after respring). My process ends like this: 

./palera.sh: line 281: 65181 Terminated: 15 "$dir"/iproxy 2222 22

Iphone 6s Ios 15.7.2
macOS High Sierra 10.13.6

**Comment by ghost** [comment]
> I still have this problem, after jb(tethered), i don't see the app(even after respring). My process ends like this:
> 
> ./palera.sh: line 281: 65181 Terminated: 15 "$dir"/iproxy 2222 22
> 
> Iphone 6s Ios 15.7.2 macOS High Sierra 10.13.6

that is normal, and it installs palera1n into tips app on ios 15, it only installs into separate app on ios 16

**Comment by ktulufhtagn** [comment]
Thanks, my bad, i had missed this thing.

**Comment by ghost** [comment]
> Thanks, my bad, i had missed this thing.

the other two people above you had real issues with ios 16 jelbrek and i never responded because i was working on fixing it still

**Comment by ghost** [comment]
@leetut @KirkLandTrip i sent out build 422 https://github.com/netsirkl64/palera1n-High-Sierra/releases gui app, if you remove the mac app of your computer and reinstall it using the latest build- and then restorerootfs under options in gui app and then rejailbreak using the updated gui app. it should fix uicache on ios 16 so it does not disappear after respring. can you pls test and let me know what happens, i do not have an ios 16 device to test on anymore.

**Comment by ghost** [comment]
Yes it works! Thank you, Mac terminal said can't restorerootfs on ios16 and iPhone froze showing verbose output, so I just restarted and ignored that

**Comment by ghost** [comment]
> Yes it works! Thank you, Mac terminal said can't restorerootfs on ios16 and iPhone froze showing verbose output, so I just restarted and ignored that

so rejailbreaking using the new build fixed palera1n on your ios 16 device using my gui? i just want to confirm this issue fixed

**Comment by ghost** [comment]
Yes

**Comment by ghost** [comment]
> Yes

thank you :heart:
